src/components/skeletonView/index.tsx

```diff
@@ -407,22 +410,66 @@ class SkeletonView extends Component<SkeletonViewProps, SkeletonState> {
       return null;
     }
 
-    const {times, timesKey, renderContent, testID} = this.props;
+    const {
+      times,
+      timesKey,
+      renderContent,
+      showContent,
+      customValue,
+      contentData,
+      template,
+      listProps,
+      size,
+      contentType,
+      hideSeparator,
+      showLastSeparator,
+      height,
+      width,
+      borderRadius,
+      circle,
+      style,
+      testID,
+      ...others
+    } = this.props;
+
+    const passedProps = {
+      showContent,
+      renderContent,
+      customValue,
+      contentData,
+      template,
+      listProps,
+      size,
+      contentType,
+      hideSeparator,
+      showLastSeparator,
+      height,
+      width,
+      borderRadius,
+      circle,
+      style,
+      testID
+    };
 
     if (times) {
-      return _.times(times, index => {
-        const key = timesKey ? `${timesKey}-${index}` : `${index}`;
-        return (
-          <SkeletonView
-            {...this.props}
-            key={key}
-            testID={`${testID}-${index}`}
-            renderContent={index === 0 ? renderContent : this.renderNothing}
-            hideSeparator={this.hideSeparator || (!this.showLastSeparator && index === times - 1)}
-            times={undefined}
-          />
-        );
-      });
+      return (
+        <View {...others}>
+          {_.times(times, index => {
+            const key = timesKey ? `${timesKey}-${index}` : `${index}`;
+            return (
+              <SkeletonView
+                modifiers={{}}
```

Comment:
Can we pass null here instead?

Reply:
It throws a TS error:
`Type 'null' is not assignable to type 'ExtractedStyle'.ts(2322)
`
I can set it to null and add a `@ts-expect-error`, wdyt?

Reply:
Where the `modifiers` prop typing comes from? 
What about passing `undefined` or is that ignored?

Reply:
Same error for `undefined` as well.
The type comes from `asBaseComponent`:
` modifiers: ReturnType<typeof Modifiers.generateModifiersStyle>;
`

Reply:
Got it, ok. Approving..